src/ripple/app/misc/ValidatorList.h

```diff
@@ -309,7 +309,8 @@ class ValidatorList
         std::function<void(PublicKey const&, bool)> func) const;
 
     static std::size_t
-    calculateQuorum (std::size_t nTrustedKeys);
+    calculateMinimumQuorum (
```

Comment:
OR... what do you think about `calculateMinimumQuorum` being private.
I'd remove `testCalculateMinimumQuorum` and move most of its checks down here:
https://github.com/ripple/rippled/pull/2093/commits/6ef1f30e4e84d8155b9d0b69666c5ed50eaec09a#diff-03bfc5ce29ad8cf375088f420df2649dR780